src/Executor.cpp

```diff
@@ -2,17 +2,19 @@
 
 #include <folly/Memory.h>
 #include <folly/futures/QueuedImmediateExecutor.h>
+#include <folly/io/IOBuf.h>
 #include "src/StackTraceUtils.h"
 #include "src/SubscriberBase.h"
 
 namespace reactivesocket {
 
 // just instantiating of the template here
 template class SubscriberBaseT<Payload>;
+template class SubscriberBaseT<folly::IOBuf>;
```

Comment:
The template is declared as extern in the header file, to prevent instantiation of the template in every compilation unit. It has to be instantiated once somewhere. I can add a new cpp file to instantiate the template there, but instead I added it here. If you have a preference where to do it, please let me know.